api/src/main/java/com/openlattice/edm/requests/EdmRequest.java

```diff
@@ -18,19 +18,19 @@
 
 package com.openlattice.edm.requests;
 
-import java.util.Set;
-import java.util.UUID;
-
-import com.openlattice.client.serialization.SerializationConstants;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.client.serialization.SerializationConstants;
+
+import java.util.Set;
+import java.util.UUID;
 
 public class EdmRequest {
-    public static enum Action {
+    public enum Action {
```

Comment:
that's definitely an option, but it will involve designing what REPLACE means in the context of a permission request or otherwise handling it being passed as transforming it to SET which feels like bad practice but the correct behavior. The best thing is probably to get rid of this class altogether since I'm pretty sure its usage is in super highly deprecated, never used, extremely broken APIs that we can probably just delete, but I was trying to steer clear of actually deleting APIs in this PR and stick to more straightforward deletes.